llvm/test/CodeGen/AArch64/stack-protector-trap-unreachable.ll

```diff
@@ -0,0 +1,34 @@
+; NOTE: Do not autogenerate, we'll lose the important NOT- check otherwise.
+;; Make sure we emit trap instructions after stack protector checks iff NoTrapAfterNoReturn is false.
```

Comment:
-NEXT checks that exclude the instruction are superior to -NOT checks, which will not fail if the output ever changes

Reply:
Noted - I'll add some -NEXT checks too. I'd like to keep the -NOT check since it makes it explicit what we're testing for.